.github/workflows/android.yml

```diff
@@ -3,6 +3,7 @@ name: Android CI
 on:
   push:
     branches: [ master ]
+  workflow_dispatch:
```

Comment:
?

Reply:
I try to test in my actions.^_^
![image](https://user-images.githubusercontent.com/41138521/183085415-7c810b52-b47d-440c-9c27-7cf3897c3dbf.png)

Reply:
dont submit it to upstream.

Reply:
Ok, sorry